stacks/go/1.4.0/devfile.yaml

```diff
@@ -0,0 +1,79 @@
+schemaVersion: 2.2.2
```

Comment:
1.x stack versions should be still using devfile 2.1.0:
```suggestion
schemaVersion: 2.1.0
```

Reply:
why?

Reply:
https://github.com/devfile/registry/pull/386#issuecomment-2117944019